src/test/phold/CMakeLists.txt

```diff
@@ -35,4 +35,19 @@ add_shadow_tests(
     LOGLEVEL info
     ARGS --pin-cpus --workers 2
     PROPERTIES RUN_SERIAL TRUE
-    CONFIGURATIONS ilibc)
\ No newline at end of file
+    CONFIGURATIONS ilibc)
+
+# Run tests with the round-robin queueing discipline (the current default is fifo).
+add_shadow_tests(
```

Comment:
Optional: is there a more focused test to which we could add an RR configuration? phold is more of a stress/integration test than a unit test.

Reply:
Ideally we'd want to use the RR configuration on a test which uses the network interface a lot, so one which reads and writes a lot of data, and one that sends data on many sockets at once per host. Phold is probably too simple of a test for this, but I figured it's easier for now to use phold rather than writing a new plugin that duplicates a lot of phold's functionality.